<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videodisplayinqtClass</class>
 <widget class="QMainWindow" name="videodisplayinqtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>videodisplayinqt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>                               视频显示</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>相机显示</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>视频显示</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="videodisplayinqt.qrc"/>
 </resources>
 <connections/>
</ui>
